.github/workflows/test.yml

```diff
@@ -47,66 +47,89 @@ jobs:
+          RPC_PORT: 18443
+          FALLBACKFEE: "0.00001"
+        volumes:
+          - bitcoind-data:/data
+      electrs:
+        image: ghcr.io/farcaster-project/containers/electrs:0.8.11
+        env:
+          NETWORK: regtest
+          DAEMON_RPC_ADDR: bitcoin-core:18443
+          ELECTRUM_RPC_PORT: 50001
+        volumes:
+          - bitcoind-data:/data
+      monerod:
+        image: ghcr.io/farcaster-project/containers/monerod:0.17.2.3
+        env:
+          NETWORK: regtest
+          MONEROD_RPC_PORT: 18081
+          OFFLINE: --offline
+          DIFFICULTY: 1
+      monero-wallet-rpc-1:
+        image: ghcr.io/farcaster-project/containers/monero-wallet-rpc:0.17.2.3
+        env:
+          MONEROD_ADDRESS: monerod:18081
+          WALLET_RPC_PORT: 18083
+      monero-wallet-rpc-2:
+        image: ghcr.io/farcaster-project/containers/monero-wallet-rpc:0.17.2.3
+        env:
+          MONEROD_ADDRESS: monerod:18081
+          WALLET_RPC_PORT: 18084
+      monero-wallet-rpc-3:
+        image: ghcr.io/farcaster-project/containers/monero-wallet-rpc:0.17.2.3
+        env:
+          MONEROD_ADDRESS: monerod:18081
+          WALLET_RPC_PORT: 18085
+
     steps:
       - uses: actions/checkout@v2
-      - name: Install dependencies
-        run: sudo apt-get install -y cmake libzmq3-dev docker-compose
-
-      - name: Compose docker services
-        run: cd tests && docker-compose up -d
 
-      - name: Install Rust stable
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-
-      - name: Correct permissions
-        run: sudo chown -R $USER tests
+      - name: Refresh cache and add apt-utils
+        run: apt-get update -y && apt-get install -y --no-install-recommends apt-utils
+      - name: Install dependencies
+        run: DEBIAN_FRONTEND=noninteractive apt-get install -y libsqlite3-dev libssl-dev libzmq3-dev pkg-config build-essential cmake
 
       - name: Rust cache
         uses: Swatinem/rust-cache@v1.3.0
         with:
           key: test-${{ matrix.test }}
 
-      - name: Cargo test
-        uses: actions-rs/cargo@v1
-        with:
-          command: test
-          args: ${{ matrix.test }} --workspace --all-targets --all-features --no-fail-fast --verbose -- --ignored --test-threads=1
-
-  functional-swap-test:
-    name: Functional swap tests
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - name: Install dependencies
-        run: sudo apt-get install -y cmake libzmq3-dev docker-compose
-
-      - name: Compose docker services
-        run: cd tests && docker-compose up -d
-
-      - name: Install Rust stable
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-
-      - name: Correct permissions
-        run: sudo chown -R $USER tests
-
-      - name: Rust cache
-        uses: Swatinem/rust-cache@v1.3.0
-
-      - name: Cargo test
-        uses: actions-rs/cargo@v1
+      - name: Cargo swap test
```

Comment:
s/Cargo swap test/Cargo functional test/ ?

Reply:
Done in 55527976931c5bc192a6352e1a44830fc5dc0f38